src/mac/LoRaMac.h

```diff
@@ -83,6 +83,18 @@ extern "C"
 #include "secure-element-nvm.h"
 #include "LoRaMacClassBNvm.h"
 
+#ifndef LORAMAC_VERSION

```

Comment:
Could you please remove the `#ifndef`/`#endif` statements?

As discussed on #1285 issue this does not look to be necessary anymore.

Reply:
Pushed in a separate commit into the PR.